packages/non-core/less/tests/top.import.less

```diff
@@ -6,4 +6,4 @@
 
 // Make sure regular CSS import doesn't make the compiler explode - this was
 // a regression from 1.1.0.3 caught in QA
-@import url("http://hello.myfonts.net/count/2c4b9d");
+@import url("./hello-myfonts.css");
```

Comment:
It seems this test was for how it handles urls that start with http:// or https://. If the old url doesn't work, it would be good to replace it with one that starts the same.